### ISSUE TITLE
scrollToItem with animation will behave unexpectedly.

### ISSUE THREAD

**Issue by PKPK-Carnage** [arnauddorgans/InfiniteLayout#2]
Thank for good work.

 I found InfiniteCollectionView's inexplicable movements when it do scrollToItem with animation.

``` swift

@IBAction func didTapScrollToOneButton(_ sender: Any) {
    self.collectionView?.scrollToItem(at: IndexPath(item: 0, section: 0), at: .centeredHorizontally, animated: true)
}

```

GIF
![scroll](https://user-images.githubusercontent.com/20692907/43998686-6314db3e-9e36-11e8-9daa-7dfeb4e5da75.gif)

Probably, I think scrollToItem's argument indexpath is not be optimized for InfiniteCollectionView.

I am glad if you can check it.

**Comment by arnauddorgans** [comment]
Hi !
Thx, i've made the fixes in 0.2.3.3
You can update it right now :)

**Comment by PKPK-Carnage** [comment]
You did a very good job!!!

Merci beaucoup :)
